### PR TITLE
New/view raw file

### PR DESCRIPTION
**Pull request by TheSubliminal**

### CODE REVIEW

File: client/src/containers/FileViewPage/index.jsx [hunk 1]
```diff
@@ -45,6 +46,14 @@ class FileViewPage extends React.Component {
     navigator.clipboard.writeText(this.filepath);
   }
 
+  handleOpenRaw() {
+    const { match } = this.props;
+    const { username, reponame, branch } = match.params;
+
+    window.location
+      .replace(`${process.env.REACT_APP_RAW_SERVER_URL}/${username}/${reponame}/${branch}/${this.filepath}?token=${localStorage.getItem('token')}`);
```
Comment (OlgaGnatenko) [inline client/src/containers/FileViewPage/index.jsx]: Maybe it's better to reference process.env in app.config.js like that: const rawServerUrl = process.env.REACT_APP_RAW_SERVER_URL;
and then to import rawServerUrl in FileViewPage ?

File: raw-server/server.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+require('dotenv').config();
+const express = require('express');
+const axios = require('axios');
+
+const app = express();
+const port = process.env.RAW_SERVER_PORT || 3002;
+const serverHost = process.env.SERVER_HOST;
```
Comment (OlgaGnatenko) [inline raw-server/server.js]: Same thing about process.env usage - should we reference it in common.config.js, and then import from common config (in server part)?

Comment (TheSubliminal) [reply raw-server/server.js]: This server consists of the single file, so don't think it's so critical in this situation.

Comment (OlgaGnatenko) [reply raw-server/server.js]: Probably yes, it's small then :)

**Pull request merged**